ChatQnA/Dockerfile.wrapper

```diff
@@ -0,0 +1,36 @@
+
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM python:3.11-slim
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends --fix-missing \
+    libgl1-mesa-glx \
+    libjemalloc-dev \
+    git
+
+RUN useradd -m -s /bin/bash user && \
+    mkdir -p /home/user && \
+    chown -R user /home/user/
+
+COPY /GenAIComps /home/user/GenAIComps
+WORKDIR /home/user/
+#RUN git clone --branch v1.0 https://github.com/opea-project/GenAIComps.git
```

Comment:
We shall get it from clone itself. This line shall not be commented out.

Reply:
Uncommented the git clone command